src/test/java/it/gov/pagopa/payment/service/qrcode/QRCodeAuthPaymentServiceTest.java

```diff
@@ -39,6 +40,7 @@ class QRCodeAuthPaymentServiceTest {
   @Mock private RewardCalculatorConnector rewardCalculatorConnector;
   @Mock private TransactionNotifierService notifierService;
   @Mock private ErrorNotifierService errorNotifierService;
+  @Mock private AuditUtilities auditUtilities;
```

Comment:
When it's a mock, let's explicitly it in the name (please could you fix also the other properties?)
```suggestion
  @Mock private AuditUtilities auditUtilitiesMock;
```